### PR TITLE
Builder lifetime

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/include/network/lan/address.rb [hunk 1]
```diff
@@ -97,6 +97,7 @@ def AddressDialog(builder:)
       # proceed with WLAN settings if appropriate, #42420
       ret = :wire if ret == :next && builder.type == "wlan"
 
+      Builtins.y2milestone("AddressDialog res: %1", ret)
```
Comment (teclator) [inline src/include/network/lan/address.rb]: `log.info is already used in the file, so will continue using it`

Comment (jreidinger) [reply src/include/network/lan/address.rb]: ah, ok. I add it mainly for debugging, but I think it makes sense to keep it.

File: src/lib/y2network/dialogs/add_interface.rb [hunk 2]
```diff
@@ -20,24 +20,48 @@ def contents
...
+        Yast::LanItems.AddNew
+        # FIXME: can be partly deleted and partly moved
+        Yast::Lan.Add
+
+        # FIXME: This is for backward compatibility only
+        # dhclient needs to set just one dhcp enabled interface to
+        # DHCLIENT_SET_DEFAULT_ROUTE=yes. Otherwise interface is selected more
+        # or less randomly (bnc#868187). However, UI is not ready for such change yet.
+        # As it could easily happen that all interfaces are set to "no" (and
+        # default route is unrecheable in such case) this explicite setup was
```
Comment (teclator) [inline src/lib/y2network/dialogs/add_interface.rb]:
```suggestion
        # default route is unreachable in such case) this explicit setup was
```

**Comment by teclator** [comment]
In general looks good to me.

**Pull request merged**